File: Data Structures & Algorithms/majority-element/submission-1.java
class Solution {
    public int majorityElement(int[] nums) {
        int c=0;
        int e=nums[0];
        for(int x:nums)
        {
            if(x==e)
            {
                c++;
            }
            else 
            {
                if(c==0)
                {
                    e=x;
                
                }
                else 
                {
                    c--;
                }
                
            }
        }
        return e;
        
    }
}

File: Data Structures & Algorithms/permutations-ii/submission-1.py
from itertools import permutations
from typing import List

class Solution:
    def permuteUnique(self, nums: List[int]) -> List[List[int]]:
        l = set()
        for p in permutations(nums):
            l.add(p)  # add the tuple, not list(p)
        return [list(p) for p in l]  # convert each tuple to a list at the end